learn/features/eth-compatibility.md

```diff
@@ -23,3 +25,51 @@ Moonbeam supports:
  - **Ecosystem Tools** (e.g., block explorers, front-end development libraries, wallets)
  - **Development Tools** (e.g., Truffle, Remix, MetaMask)
  - **Ethereum Tokens via Bridges** (e.g., token movement, state visibility, message passing)
+
+## Frontier {: #frontier }
+
+[Frontier](https://paritytech.github.io/frontier/){target=_blank} is an Ethereum compatibility layer for Substrate. The goal of Frontier is to allow standard Ethereum DApps to run without modification on Substrate-based chains. Frontier makes this possible by  offering some Substrate pallets that can be plugged into a Substrate runtime. The following pallets can be used independently, as needed, or collectively depending on the chain's desired functionality:
+
+- **[EVM pallet](#evm-pallet){target=_blank}** - handles EVM execution
+- **[Ethereum pallet](#ethereum-pallet){target=_blank}** - is responsible for storing block data and provides RPC compatibility
+- **[Base fee pallet](#base-fee-pallet){target=_blank}** - adds support for EIP-1559 transactions and handles base fee calculations
+- **Dynamic fee pallet** - calculates the dynamic minimum gas price
+
+Moonbeam uses the EVM, Ethereum, and base fee pallets to achieve full Ethereum compatibility. Moonbeam does not use the dynamic fee pallet.
+
+### EVM Pallet {: #evm-pallet }
+
+The [EVM pallet](https://paritytech.github.io/frontier/frame/evm.html){target=_blank} implements a sandboxed virtual stack machine and uses the [SputnikVM](https://github.com/rust-blockchain/evm){target=_blank} as the underlying EVM engine. 
+
+The EVM executes Ethereum smart contract bytecode, of which is typically written in a language like Solidity, and then compiles it to EVM bytecode. The goal of the EVM pallet is to emulate the functionality of executing smart contracts on Ethereum within the Substrate runtime. As such, it allows existing EVM code to be executed in Substrate-based blockchains. 
+
+Inside of the EVM are standard H160 Ethereum-style accounts, and they have associated data such as the balance and nonce. These accounts are configured to be mapped to a Substrate account type. Since Moonbeam also uses H160 Ethereum-style accounts, Moonbeam has configured the Substrate account type to be a non-standard H160 account. For more information on Moonbeam's account system, please refer to the [Unified Accounts](/learn/features/unified-accounts/){target=_blank} page.
+
+To execute within the EVM, a mapped Substrate account can call the EVM pallet to deposit or withdraw balance from the Substrate-base currency into a different balance managed and used by the EVM pallet. Once a balance exists, smart contracts can be created and interacted with. Other pallets can also be configured to execute within the EVM. For Moonbeam, pallet Ethereum has been configured to have sole responsibility of EVM execution.
```

Comment:
Just above you said we don't need Substrate accounts, not sure if this is confusing here

Reply:
where do I say that?